cypress-shared/support/commands.js

```diff
@@ -193,3 +193,52 @@ Cypress.Commands.add('openStoreFront', (login = false) => {
 
   scroll()
 })
+
+Cypress.Commands.add('addNewLocation', (country, PostalCode) => {
+  cy.openStoreFront()
+  cy.get(selectors.addressContainer).click()
+  cy.get(selectors.countryDropdown).select(country)
+  cy.get(selectors.addressInputContainer)
+    .first()
+    .clear()
+    .should('be.visible')
+    .type(PostalCode)
+  cy.get(selectors.Address)
+    .contains('Address Line 1')
+    .parent()
+    .within(() => {
+      cy.get("input[type='text']")
+        .clear()
+        .type('1481 Maple View Dr,Promona,CA,USA')
+    })
+  cy.get(selectors.Address)
+    .contains('City')
+    .parent()
+    .within(() => {
+      cy.get("input[type='text']").clear().type('Promona')
+    })
+  cy.get(selectors.SelectState).select('California')
+  cy.waitForGraphql('address', selectors.SaveButton)
+  cy.once('uncaught:exception', () => false)
```

Comment:
Do we really need this ?

Reply:
yes it is failing sometimes as we noticed @saravananvenkatesan / @sumanraj123  added this